modules/t/publicationAdaptor.t

```diff
@@ -77,6 +77,21 @@ $va->db->include_failed_variations(1);
 my $varfs = $va->fetch_all_by_publication($pubs->[0]);
 ok(scalar(@{$varfs}) ==2,   "variation count by publication - inc fails");
 
+## store NULL authors
```

Comment:
Suggest comment modified to say what testing e.g. authors stored as NULL when not  provided. 
Includes "" and 0.